test/verify/check-static-login

```diff
@@ -955,6 +955,68 @@ matchrule = <SUBJECT>^DC=LAN,DC=COCKPIT,CN=alice$
         # by IPv6 address and port
         check_server("[::1]:22", expect_fp_ack=True)
 
+    # check cockpit-beiboot as replacement of cockpit-ssh on the login page
+    # once that becomes the default, TestMultiMachine* and the other TestLogin* cover this
+    @testlib.skipImage("needs pybridge", "rhel-8*", "centos-8*")
+    # enable this once our cockpit/ws container can beiboot
+    @testlib.skipOstree("client setup does not work with ws container")
+    def testLoginSshBeiboot(self):
```

Comment:
Can you add a test scenario to find out what happens if you specify a username *and* a same/different `user@host`?

Reply:
You mean writing `user@host` into the `Connect to:` field? As I said above, that has never been intended to work, but I'll add it to pin down the behaviour.

Reply:
Done.